debiandesktop_ec2.tf

```diff
@@ -56,7 +56,8 @@ resource "aws_instance" "debiandesktop" {
   }
   subnet_id = aws_subnet.operations.id
   tags = {
-    Name = format("DebianDesktop%d", count.index)
+    Name             = format("DebianDesktop%d", count.index)
+    "Publish Egress" = var.publish_egress_ip_addresses
```

Comment:
Will this value massage to the `True` or `False` that was in use before? I'm concerned this will interpolate to a string version of `true` or `false` which doesn't quite match up.

Reply:
You are good!  After this PR is merged, I was planning to modify the lambda code [here](https://github.com/cisagov/publish-egress-ip-lambda/blob/bfb38499afdb29ac622f687a364088e8bf19e119/src/lambda_handler.py#L97-L110) to account for this change from `True` to `true`.